lib/lambda_ethereum_consensus/state_transition/math.ex

```diff
@@ -0,0 +1,42 @@
+defmodule LambdaEthereumConsensus.StateTransition.Math do
+  @moduledoc """
+  Math functions
+  """
+
+  @spec uint_to_bytes(SszTypes.uint64()) :: binary
+  def uint_to_bytes(value) when is_integer(value) do
+    byte_size = calculate_byte_size(value)
```

Comment:
The problem with `get_beacon_proposer_index` is probably here. Elixir integers are big-nums, so their size is not fixed. Because of that, we can't use a single `uint_to_bytes` function, and need multiple `uintN_to_bytes` functions. See:
https://github.com/lambdaclass/lambda_ethereum_consensus/blob/cbb9dc206e8602dfd27636ed35910c33e4933b52/lib/lambda_ethereum_consensus/state_transition/misc.ex#L121-L132

Reply:
Great, thanks. I'll try fix that part. 
Then will it be good to implement that in `accessor.ex` as a private function, or make a separate .ex file for that like `math.ex`?

Reply:
You can add them in `misc.ex` or make the existing ones public. We'll probably reorganize all of these modules anyway.

Reply:
Got it!